ckan/lib/base.py

```diff
@@ -124,12 +127,17 @@ def _allow_caching(cache_force: Optional[bool] = None):
     elif request.args.get('__no_cache__'):
         allow_cache = False
     # Don't cache if caching is not enabled in config
-    elif not config.get('ckan.cache_enabled'):
+    elif not h.asbool(config.get('ckan.cache_enabled', True)):
```

Comment:
config declaration enforces the type of this value, there's no need for `asbool`

Reply:
will cleanup